src/loaders/JsonImageInfoLoader.ts

```diff
@@ -145,11 +154,15 @@ class JsonImageInfoLoader extends ThreadableVolumeLoader {
   async loadDims(loadSpec: LoadSpec): Promise<VolumeDims[]> {
     const jsonInfo = await this.getJsonImageInfo(loadSpec.time);
 
-    const d = new VolumeDims();
-    d.shape = [jsonInfo.times || 1, jsonInfo.channels, jsonInfo.tiles, jsonInfo.tile_height, jsonInfo.tile_width];
-    d.spacing = [1, 1, jsonInfo.pixel_size_z, jsonInfo.pixel_size_y, jsonInfo.pixel_size_x];
-    d.spaceUnit = jsonInfo.pixel_size_unit || "μm";
-    d.dataType = "uint8";
+    const [px, py, pz] = rescalePixelSize(jsonInfo);
+
+    const d: VolumeDims = {
+      shape: [jsonInfo.times || 1, jsonInfo.channels, jsonInfo.tiles, jsonInfo.tile_height, jsonInfo.tile_width],
+      spacing: [1, 1, pz, py, px],
+      spaceUnit: jsonInfo.pixel_size_unit || "μm",
+      dataType: "uint8",
+      timeUnit: jsonInfo.time_unit || "s",
```

Comment:
some of these might want to be `??` rather than `||` too, though `jsonInfo.times` is at least a bit subtle because `0` is also a problematic value

Reply:
Yea I think maybe we want to intentionally always have at least numT=1.  What do you think about handling empty string for time_unit and pixel_size_unit?

Reply:
I think it seems possible that a user might want to have no space/time unit, and would express that with an empty string? Though I know unit strings are read and interpreted in a couple places (@ShrimpCryptid's unit formatter, my unit abbreviator), and can't necessarily predict what either would do with those.

Reply:
okay `??` it is then. at least for the unit strings.